libraries/helpers.rb

```diff
@@ -354,7 +354,7 @@ def url_for(pkg, base_url = 'ftp://ftp.software.ibm.com/aix/freeSoftware/aixtool
           'xft' => { version: '2.1.6-5', rpm: 'xft/xft-2.1.6-5.aix5.1.ppc.rpm' },
           'xpdf' => { version: '1.00-1', rpm: 'xpdf/xpdf-1.00-1.aix4.3.ppc.rpm' },
           'xpm-devel' => { version: '3.4k-8', rpm: 'xpm/xpm-devel-3.4k-8.aix6.1.ppc.rpm' },
-          'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
+          # 'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
```

Comment:
Duplicate keys removed as requested  Is there something else wrong here @chmod666org or @tas50 .  I'll fix whatever is necessary, my fork has been working well on my AIX systems.  Is anything wrong